src/clients/jobManagerWrapper.ts

```diff
@@ -46,35 +48,34 @@ export class JobManagerWrapper extends JobManagerClient {
     this.jobDomain = config.get<string>('externalClientsConfig.clientsUrls.jobManager.jobDomain');
   }
 
-  /**
-   * @deprecated The method should not be used
-   */
-  public async create(data: IWorkerInput): Promise<ICreateJobResponse> {
+  @withSpanAsyncV4
+  public async createExport(data: IWorkerExportInput): Promise<ICreateExportJobResponse> {
     const expirationDate = new Date();
     expirationDate.setDate(expirationDate.getDate() + this.expirationDays);
 
-    const createJobRequest: CreateJobBody = {
+    const jobParameters: IJobExportParameters = {
```

Comment:
add span id / context to job parameters as is done with cache-seeder in overseer (for later use)